it/src/test/java/org/eclipse/jkube/integrationtests/vertx/VertxOcITCase.java

```diff
@@ -109,6 +109,25 @@ void ocApply() throws Exception {
 
   @Test
   @Order(4)
+  @DisplayName("oc:log, should retrieve log")
+  void ocLog() throws Exception {
+    // Given
+    final Properties properties = new Properties();
```

Comment:
Everything looks fine, but whenever you introduce some new class in your code. You need to tell the compiler where it is. That is done using class's fully qualified name(check import statements on top of this file). I think you forgot to tell compiler about it. Just add these lines in import statements(by IDE you can also do it via `Alt + L`:
```
import java.io.ByteArrayOutputStream;
import java.io.PrintStream;
import java.nio.charset.StandardCharsets;
import java.util.Properties;
import org.eclipse.jkube.integrationtests.maven.MavenUtils;

import static org.hamcrest.Matchers.stringContainsInOrder;
```